settings/_settings.global.scss

```diff
@@ -2,17 +2,23 @@
 //   #GLOBAL
 //   ======================================================================== */
 
-// Project-level settings and setup.
+// The global settings file contains any project-wide variables.
+
 
 // Base typographical styles and baseline grid. You need to define these values
 // in pixels: inuitcss will convert them to more appropriate units.
+
 $inuit-global-font-size:    16px !default;
 $inuit-global-line-height:  24px !default;
 
+
 // Do we need to support IE8?
+
 $inuit-oldIE: false !default;
 
+
 // Standardise some UI treatments.
+
 $inuit-global-radius:       3px !default;
```

Comment:
I personally don’t think we should define a global radius if we don’t apply it to various UI components. Since the aim is to not provide a UI library I’d say this is up to the user to set, and therefore should be removed from inuit.